src/vivarium/framework/population/population_view.py

```diff
@@ -155,22 +158,28 @@ def get(self, index: pd.Index[int], query: str = "") -> pd.DataFrame:
         Raises
         ------
         PopulationError
-            If this view has access to columns that have not yet been created
-            and this method is called. If you see this error, you should
-            request a subview with the columns you need read access to.
+            If the requested attributes are not a proper subset of this view's columns.
 
-        See Also
-        --------
-        :meth:`subview <PopulationView.subview>`
         """
 
+        if isinstance(attributes, str):
+            attributes = [attributes]
+
+        if set(attributes) - set(self.columns):
```

Comment:
this is a similar check/raise as is being done in subview (which we will be removing next)